promgen/templates/promgen/alert_list.html

```diff
@@ -4,7 +4,12 @@
 
 {% breadcrumb label="Alerts" %}
 
-{% include 'promgen/pagination_short.html' %}
+<form class="navbar-form navbar-left" action="/alert">
+    <div class="form-group">
+        <input name="search" type="text" value="{{ search }}" class="form-control" placeholder="Service, Project">
```

Comment:
I think you can write this as `value="{{ request.GET.search }}"`  and then you do not have to implement `get_context_data` in the view

Reply:
fixed